public/app/plugins/datasource/prometheus/datasource.ts

```diff
@@ -44,6 +44,7 @@ import { PrometheusVariableSupport } from './variables';
 import PrometheusMetricFindQuery from './metric_find_query';
 
 export const ANNOTATION_QUERY_STEP_DEFAULT = '60s';
+const EXEMPLARS_NOT_SUPPORTED = 'Exemplars are not supported in this version of prometheus.';
```

Comment:
I'm thinking of a case when Prometheus supports exemplars but the endpoint fails some other reason and this message could be somehow misleading. Would it make sense to make it more generic, e.g. "Exemplars for this data source are not available."?